packages/js/product-editor/src/components/custom-fields/edit-modal/edit-modal.tsx

```diff
@@ -71,6 +73,13 @@ export function EditModal( {
 		}
 
 		onUpdate( customField );
+
+		recordEvent( 'product_custom_fields_update_button_click', {
+			source: TRACKS_SOURCE,
+			custom_field_id: customField.id,
```

Comment:
I think that we don't need to track the `custom_field_id`. What do you think about removing it? If we want to track that value, I wouldn't send an `undefined` value as the `custom_field_id` when updating an unsaved custom field.